package.json

```diff
@@ -53,8 +53,9 @@
     "pc:install": "python3 pre-commit.pyz install -t pre-push -t pre-commit",
     "pc:lint": "python3 pre-commit.pyz run --all-files",
     "i18n": "pnpm i18n:create-locales-list && pnpm i18n:get-translations && pnpm i18n:update-locales",
-    "i18n:copy-test-locales": "cp -r test/locales/* src/locales/",
-    "i18n:no-get": "pnpm i18n:create-locales-list && pnpm i18n:update-locales",
+    "i18n:watch-en": "pnpm i18n:get-translations --en-only --watch",
```

Comment:
In a freshly-cloned repository, if you try to run `pnpm test:playwright`, it will still raise an error because there will be no `en.json` yet.
I updated the `test:playwright:local` command on line 40 to `DISABLE_SENTRY=true pnpm i18n:copy-test-locales && playwright test -c test/playwright` to fix it locally and it worked, but I'm not sure if it'll work in the CI.

Reply:
In the CI, there is a step to [set up the `en.json` file](https://github.com/WordPress/openverse-frontend/pull/1949/files#diff-8fc5d4267ec6adb8c520d4a3cd87dd2c3026b516bc7fd7ba336dc697733c775dR35). That's why the CI workflow is passing for the PR.

Reply:
This works well!
I was going to say that if you run `pnpm test:playwright` without running `pnpm install` previously, you would still get the `no en.json found` error. But I guess you wouldn't be able to run the tests anyway because you wouldn't have any dependencies installed 🤦